codegen/templates/webhooks/_namespace.py.jinja

```diff
@@ -131,7 +132,7 @@ class WebhookNamespace:
         return type_validate_python(Event, payload)
 
     @staticmethod
-    def normalize_payload(payload: Union[GitHubModel, dict[str, Any]]) -> str:
+    def normalize_payload(payload: Union[GitHubModel, Mapping[str, Any]]) -> str:
```

Comment:
json can not dumps `Mapping`. This should be dict only (including below).

Reply:
The value is passed to `jsonable()` which accepts any iterable.

Reply:
```python
payload = model_dump(payload) if isinstance(payload, GitHubModel) else payload
```

It seems only Pydantic models are turned into dict.

Reply:
Hrm, I got something mixed up then. We can use `jsonable()` here too?

Reply:
Ah, no, I found it: `GithubCore._request()` and `GithubCore._arequest()` pass the payload to `to_jsonable_python()`:

https://github.com/yanyongyu/githubkit/blob/8f44dc00407cb2516b9ea6d017b7519a9d510317/githubkit/core.py#L287

Reply:
Actually, we can't. Mapping is _not_, in fact, supported, by `to_jsonable_python`. 🤦 

I'm reverting these mapping changes here, mea culpa.

Reply:
I also reverted the change from `dict` to `Mapping` for GraphQL variables as they are inserted into a JSON object too.